templates/base.html

```diff
@@ -0,0 +1,137 @@
+{% load static %}
+{% load widget_tweaks %}
+<!DOCTYPE html>
+<html>
+<head>
+
+	<meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	
+	<!-- PLUG INS -->
+	<link rel="stylesheet" type="text/css" href="{% static 'assets/node_modules/select2/dist/css/select2.min.css' %}">
+	<link rel="stylesheet" type="text/css" href="{% static 'assets/node_modules/animate.css/animate.min.css' %}">
+	<link rel="stylesheet" type="text/css" href="{% static 'assets/node_modules/ionicons/css/ionicons.min.css' %}">
+	<link rel="stylesheet" type="text/css" href="{% static 'assets/node_modules/bootstrap/dist/css/bootstrap-reboot.min.css' %}">
+	<link rel="stylesheet" type="text/css" href="{% static 'assets/node_modules/bootstrap/dist/css/bootstrap.min.css' %}">
+	<link rel="stylesheet" type="text/css" href="{% static 'assets/css/style.css' %}">
+	<!-- PLUG INS -->
+
+	<title>Marketplace</title>
+</head>
+<body >
+	<nav class="navbar navbar-expand-lg main-nav">
+		<div class="container">
+			<a href="{% url 'account:home' %}" class="navbar-brand">
+				<img src="{% static 'assets/images/logo.svg' %}" class="logo">
```

Comment:
this is so wrong, why? because you changed the configuration that i set. your code converts to this.
`/static/assets/images/logo.svg`